README.md

```diff
@@ -2,3 +2,7 @@
 Dotychczas ustawa o podatku dochodowym od osób fizycznych w połączeniu z ustawą o zryczałtowanym podatku dochodowym liczyły ponad 500 stron. Była archaiczna, nieczytelna, przypominała śmietnik - każdy kolejny rząd, każdy kolejny premier i minister finansów wprowadzał tam swoje pomysły, jednak nikt nie był skory zająć się błędami poprzedników. Powstał bałagan, który posprzątać mogła tylko kompleksowa zmiana. Ustawy tworzą ze sobą sieć naczyń połączonych - każda wpływa na trzy kolejne. Celem prezentowanych trzech nowych ustaw było nie tylko obniżenie podatków, ale również ograniczenie tych połączeń, uproszczenie systematyki, a także racjonalizacja przepisów i pojęć. Współczesne przepisy są nieczytelne nie tylko dla zwykłych obywateli - często były niezrozumiałe również dla prawników. Pozwalały na szeroką dozę interpretacji, wymagały nieustannych zmian i komentarzy, a także wiążących objaśnień ze strony ministerstwa. Chaos był tak wielki i rozległy nie tylko przez objętość regulacji, ale także przez niestaranne, częste i nieprzemyślane zmiany w obrębie tego wielkiego bałaganu.
 
 Podstawowe założenie regulacji to niskie i proste podatki, automatyzacja rozliczeń, zachęcanie do inwestycji i przedsiębiorczości, a także wsparcie młodych i seniorów na rynku pracy.
+
+## Jak używać Github i współtworzyć ustawę o PIT
+
+[Kliknij tutaj](CONTRIBUTING.md), aby wyświetlić plik, który pomoże ci zrozumieć jak działa platforma Github oraz jak z niej korzystać, aby pomóc w rozwoju projektu.
```

Comment:
W tym akapicie brakuje mi:
1) Jasnego określenia jaki charakter ma ten projekt? Czy jest rzeczywiscie otwarto źródłowy i im szersza grupa zaangażowanych osób tym lepiej? Czy może jest to projekt ustawy stworzony i rozwijany przez zamkniętą grupę osób, a GitHub to jedynie sposób na publikację i promocję?
2) W aktualnej formie jest to jedynie wskazówka techniczna. Brakuje słów zachęty do współtworzenia typu `Zawsze doceniamy wkład społeczności. (...)`.

Reply:
Jedynki sam nie opiszę, dwójka dodana.